pkg/kubelet/rkt/rkt.go

```diff
@@ -1108,6 +1108,18 @@ func (r *Runtime) getSelinuxContext(opt *v1.SELinuxOptions) (string, error) {
 	return strings.Join(ctx, ":"), nil
 }
 
+// From the generateName or the podName return a basename for improve the logging with the Journal
+// journalctl -t podBaseName
+func constructSyslogIdentifier(generateName string, podName string) string {
```

Comment:
Can you explain here why not just returning podName? I assume it's because `journalctl -t` doesn't support a pattern?